cli/command/stack/common.go

```diff
@@ -36,7 +42,7 @@ func getServices(
 ) ([]swarm.Service, error) {
 	return apiclient.ServiceList(
 		ctx,
-		types.ServiceListOptions{Filters: getStackFilter(namespace)})
+		types.ServiceListOptions{Filters: getServiceFilter(namespace)})
```

Comment:
Same here; is this adding new filter options to the `docker stack ls` / `docker stack services` and `docker stack ps` commands?